src/ManifoldSampling.jl

```diff
@@ -40,15 +40,22 @@ Return a random sample point on a manifold from a belief represented by coordina
 Notes
 
 """
-function samplePoint(M::AbstractManifold, sbelief, p, basis, retraction_method::AbstractRetractionMethod=ExponentialRetraction())
+function samplePoint(M::AbstractManifold, sbelief, p, basis::AbstractBasis, retraction_method::AbstractRetractionMethod=ExponentialRetraction())
   X = sampleTangent(M, sbelief, p, basis)
   return retract(M, p, X, retraction_method)
 end
 function samplePoint(M::AbstractGroupManifold, sbelief, p=identity_element(M), retraction_method::AbstractRetractionMethod=ExponentialRetraction())
   X = sampleTangent(M, sbelief, p)
   return retract(M, p, X, retraction_method)
 end
+
+function samplePoint(M::AbstractGroupManifold, sbelief::ManifoldKernelDensity, p=mean(sbelief), retraction_method::AbstractRetractionMethod=ExponentialRetraction())
```

Comment:
Just a breadcrumb, new numerical failure might stem from this change. I can’t test now though.

Reply:
Yes. This is the issue. MKD is not calculated around the mean but identity element, so should use `identity_element`
I’ll fix if i get proper signal.